### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by r3dw00dy** [hldh214/lok_bot#26] [Q&A] [answered]

**Comment by EichmannT** [comment, marked as answer]
Hi. You need to install Python on Windows, the download Pipenv. You don't specifically need Git installed if you download the code to your C:.
I integrated Python into my CMD and used that. Go to the LOKBOT folder and install pre-reqs for the code.
# pip install requirements.txt 
This pulls all the extra libraries you will need in Python to run the lokbot. Then it's a case of running it.
python -m Lokbot %youraccesscode%
You need to open League of Kingdoms in a browser, hit F12, start a network trace then logon and get your access_token from the Header in the trace.
It's a bit complicated. If you work in IT then you'll be able to fumble around and get it up and running in 30-60 minutes depending on how much code you want to read.
If you leave it open over night then the access_token will expire and you'll get an auth error and generally your LOK account will get banned. So don't test this on your main account. Create a new google email address and test using that.
I stopped the keep alive thread in the app.py and tweaked the loops a bit and my accounts stopped getting banned, but then the bot wasn't automated... so you'll need to find a balance between a fully auto bot and what the LOK devs are doing to catch your bot usage and ban your account.
Good luck.
